tests_tf/test_attacks.py

```diff
@@ -158,6 +158,25 @@ def test_generate_np_gives_adversarial_example_l1(self):
     def test_generate_np_gives_adversarial_example_l2(self):
         self.help_generate_np_gives_adversarial_example(2)
 
+    def test_targeted_generate_np_gives_adversarial_example(self):
+        x_val = np.random.rand(100, 2)
+        x_val = np.array(x_val, dtype=np.float32)
+        random_labs = np.random.random_integers(0,1,100)
```

Comment:
I think we already had a discussion about this, but it seems like this test does not have a seed for np.random. It might become flaky?